scenarios/zome_call_single_value/README.md

```diff
@@ -8,5 +8,5 @@ function is not calling into Holochain other otherwise doing any work.
 ### Suggested command
 
 ```bash
-RUST_LOG=info cargo run --package zome_call_single_value -- --connection-string ws://localhost:8888 --duration 300
+RUST_LOG=warn cargo run --package zome_call_single_value -- --connection-string ws://localhost:8888 --duration 300
```

Comment:
Don't feel strongly about this but when running locally, I'm generally happy with more information. But at info I usually end up tuning this anyway to get the modules I want logs from, so the default is use a placeholder anyway :)

Reply:
I absolutely don't care what's there. I just copied this across. I'll change it again.